.github/workflows/deploy-frontend-azurepreview.yml

```
@@ -48,6 +48,16 @@ jobs:
         with:
           fetch-depth: 0
 
+      - name: Debug checkout        # ← add this temporarily
+        run: |
```

actionlint (1.7.12)

[error] "github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)